.github/workflows/ci-test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        os: ['ubuntu-latest', 'macos-latest']
```

Comment:
Does clingraph not work properly on windows currently? Why are we removing it as a ci target?

Reply:
To install graphviz you need to download a binary file so in clingraph we also didnt test in windows